web/elm/src/Api/Endpoints.elm

```diff
@@ -118,10 +118,10 @@ toPath endpoint =
             basePath ++ [ "jobs" ]
 
         Job id subEndpoint ->
-            pipelinePath id ++ [ "jobs", id.jobName ] ++ jobEndpointToPath subEndpoint
+            pipelinePath id.pipelineId ++ [ "jobs", id.jobName ] ++ jobEndpointToPath subEndpoint
```

Comment:
Seems like an existing convention but it reads weird to access properties on id

Reply:
Yeah, perhaps a bit weird to read. The `Identifier` types aren't (necessarily) the same thing as a database ID, though - they're just some way to uniquely identify entities. The exception is now `PipelineIdentifier`, which is uniquely identified by the database ID - but e.g. `JobIdentifier` is identified by the `pipelineId` and `jobName`, since that's what it takes to uniquely specify a job

Reply:
It is really weird actually :p but we can probably improve it in future